example-scripts/voctolight/voctolight.py

```diff
@@ -100,5 +80,8 @@ def start_connection(tally_handler):
                 continue
     finally:
         print('cleanup')
-        if DO_GPIO:
-            GPIO.cleanup()
+        del plugin
```

Comment:
Is that actually necessary? This is `main()`, so the program will terminate after this line anyway. I've never seen such cleanup routines in python and I guess whoever works on this next will probably not maintain this, so I'd suggest to just remove it, unless there is a specific reason for it.

Reply:
You tell me? 😄 

Removed.